code/game/gamemodes/cult/runes.dm

```diff
@@ -1073,7 +1108,7 @@ var/list/sacrificed = list()
 	"<span class='warning'>You are blinded by the flash of red light! After you're able to see again, you see that you are now wearing a set of armor.</span>")
 
 	if(istype(user, /mob/living/simple_animal/construct))
-		var/mob/living/simple_animal/construct/C = user
+		var/mob/living/C = user

```

Comment:
This change seems pointless, and slightly misleading. Due to the typecheck above, C _cannot_ be anything other than a construct, surely?

Reply:
Yup, thats a leftover from a previous change where I initially used the armor proc to spawn the construct